docker/swarm/README.md

```diff
@@ -0,0 +1,19 @@
+# Swarm 
+
+Docker image based on nuxeo jenkins public slave that have jenkins-swarm-client integrated
```

Comment:
`nuxeo jenkins public slave` ?
jenkins-swarm-client deserves links and pointers...